evm-contracts/src/v0.6/dev/FluxAggregator.sol

```diff
@@ -527,10 +533,40 @@ contract FluxAggregator is AggregatorInterface, Owned {
    * @notice called through LINK's transferAndCall to update available funds
    * in the same transaction as the funds were transfered to the aggregator
    */
-  function onTokenTransfer(address, uint256, bytes memory) public {
+  function onTokenTransfer(address, uint256, bytes memory)
+    public
+  {
     updateAvailableFunds();
   }
 
+  /**
+   * @notice helper method for oracles to retrieve their necessary info in a
+   * single call.
+   */
+  function oracleRoundState()
```

Comment:
We also need to think about how this will be used with whitelisted contracts because this is essentially a way to bypass that if we don't override it.